libbeat/docs/loggingconfig.asciidoc

```diff
@@ -43,12 +43,12 @@ You can specify the following options in the `logging` section of the +{beatname
 
 ===== to_syslog
 
-Sends all logging output to syslog if enabled. For non-Windows systems, the default
-value is true.
+If enabled, sends all logging output to syslog (on non-Windows systems only). The default
```

Comment:
I think we can leave out the "(on non-Windows systems only)" part. It's not common to have a syslog server when running on Windows, but it's technically possible.